docs/data/migration/migration-pickers-v5/migration-pickers-v5.md

```diff
@@ -160,9 +160,14 @@ The picker components no longer have a keyboard view to render the input inside
 At some point, the mobile pickers should have a prop allowing to have an editable field without opening the modal.
 :::
 
-### ✅ Rename `shouldDisableTime` prop
+### ✅ Rename or refactor `shouldDisableTime` prop
 
-The `shouldDisableTime` prop signature has been changed. Either rename the prop to `shouldDisableClock` or refactor usage.
+The `shouldDisableTime` prop signature has been changed.
+Previously it did receive `value` as a `number` of hours, minutes, or seconds. Now it will receive the date object (based on the used adapter).
+It was done to allow for more powerful usage as well as enable the introduction of different time selection views.
```

Comment:
```suggestion
This was done to allow for more powerful usage as well as enable the introduction of different time selection views.
```

```suggestion
This will allow more powerful usage and will enable the introduction of different time selection views.
```